### DISCUSSION TITLE
how to use https in code server which use docker to employment

### DISCUSSION THREAD

**Discussion by weidongzhou1994** [coder/code-server#3847] [Q&A]
as title

**Comment by jsjoeio** [comment]
Hmm...probably something we want to add a guide on. Guessing you've already looking in the docs (see [here](https://coder.com/docs/code-server/v3.10.2/guide#https-and-self-signed-certificates)).

I have less experience with running code-server in Docker but found [this S/O question](https://stackoverflow.com/a/50810196), which would then mean our docs about [setting up Let's Encrypt with NGINX](https://coder.com/docs/code-server/v3.10.2/guide#using-lets-encrypt-with-nginx) might help, but maybe not. Calling on @bpmct who has more knowledge here.
